packages/scaffold-ui/src/views/w3m-connecting-wc-view/index.ts

```diff
@@ -33,7 +30,11 @@ export class W3mConnectingWcView extends LitElement {
   public constructor() {
     super()
     this.initializeConnection()
-    this.interval = setInterval(this.initializeConnection.bind(this), ConstantsUtil.TEN_SEC_MS)
+    /*
+     * For 15minutes, the relay handles retries and reconnections internally.
+     * We re-initialize the connection after that
+     */
+    this.interval = setInterval(this.initializeConnection.bind(this), 15 * 60 * 1000)
```

Comment:
This is the Sign SDK that handles this, and I would be concerned that depending on this behavior is dangerous. E.g. if the SDK is updated to a version w/o this